command/shell.go

```diff
@@ -34,6 +37,21 @@ func (command *ShellCommand) Execute([]string) error {
 	}
 	fmt.Printf("Type 'exit' to exit. Use ctrl-c to abort running commands.\n\n")
 
+	var filters []*manager.TagFilter
+	filters = append(filters, &manager.TagFilter{
+		Key:    "platform",
+		Values: []string{"windows"},
+	})
+
+	windowsTargets, err := m.FilterInstances(targets, filters)
+	if ( len(targets) != len(windowsTargets) ) {
+		errors.New("Targets: Cannot mix WIndows and Linux targets")
+	}
+	if (len(windowsTargets) > 0){
+		fmt.Printf("Windows Targets: %d \n\n", len(windowsTargets) )
```

Comment:
We already output a list of the instance ID's [here](https://github.com/chrisdpa-tvx/ssm-sh/blob/master/command/utils.go#L60), not sure we need to print the number of windows instances specifically (especially since we can only target one at a time)?

Reply:
Agreed